### DISCUSSION TITLE
8U*** Transmitter Error No mac address! On Android 11

### DISCUSSION THREAD

**Discussion by ailuirazal** [NightscoutFoundation/xDrip#2349] [Q&A]
Hello, 
I did research the issue I'm having with xDrip before but there were related to Android 12.
Yesterday as I was very busy I didn't see that xDrip lost connection with the Dexcom G6 transmitter. It took me about 7h to get home and restart the sensor during which I had no readings. The sensor and the transmitter are fairly young (I've started both sensor and transmitter on 8th September 2022) so that's way I just don't understand  why on Monday they worked while on Tuesday they've stopped.
I updated the xDrip version, I unpaired every device I ever paired on my phone, uninstall the app, reinstall it (installed it on my mother phone as well to test the transmitter) but I still get the error : **"No transmitter mac address!"** 
Since I'm in Romania I don't have access to the official Dexcom App to test the transmitter.
The current version of xDrip:
xDrip-plus-20220914-e7be25a
Android version:  11
phone: One plus Nord 

![IMG_20220921_121922](https://user-images.githubusercontent.com/42743852/191563349-b90e9940-f849-4113-bba7-68aff3e07f2f.jpg)
![Screenshot_20220921-105954](https://user-images.githubusercontent.com/42743852/191563835-4dca4911-999a-4609-86c1-96d62639cdc9.jpg)
![Screenshot_20220921-095848](https://user-images.githubusercontent.com/42743852/191563894-3ec6cabb-319d-4303-81bb-32f0f5962a36.jpg)

**Comment by Navid200** [comment]
Please post a screenshot of the following page:
Settings &#8722;> G5/G6 Debug Settings

**Reply by ailuirazal** [reply]
Hello Navid200,
Sorry for the late reply I haven't seen the email. Here is the requested print screen:
![image](https://user-images.githubusercontent.com/42743852/191685024-432e9efc-5b3c-4e38-8ce1-bb5ec48b3ec1.png)

**Reply by Navid200** [reply]
Those settings are not correct.
Please have a look at the following and adjust everything that is different.
https://navid200.github.io/xDrip/docs/G6-Recommended-Settings

After you have corrected all the xDrip and Android settings (included in the linked document), restart the phone.  Bring up xDrip.  Wait 10 minutes.  Then, please post a screenshot of the system status page.

**Reply by ailuirazal** [reply]
Hey Navid, 

Thank you for the detailed explanations. I did correct the settings, restarted the Phone, waited the 10 minutes and unsurprisingly I'm getting the same error. Below are the requested screenshots. I think the transmitter isn't working as expected and needs to be replaced.

![image](https://user-images.githubusercontent.com/42743852/191751768-e138b5e1-a1ee-406a-a3ef-6e41c54b2f0d.png)
![image](https://user-images.githubusercontent.com/42743852/191751930-13c7f782-3acb-4502-bc35-3d9bb65d59a5.png)
![image](https://user-images.githubusercontent.com/42743852/191752038-610534e9-6c5c-49ff-a3cb-acf67bd87a15.png)

**Reply by Navid200** [reply]
It wish you hadn't uninstalled the app.  You could have seen the transmitter battery logs to figure out if it is possibly getting too low as explained here:  https://navid200.github.io/xDrip/docs/Battery-condition

But, it is very likely all thrown out now after a reinstall.
I wish reinstall wasn't so popular.
  
You can use a scanner to see if you can see the transmitter:  https://navid200.github.io/xDrip/docs/Bluetooth-Scanner

**Reply by ailuirazal** [reply]
Yeah, I'm sorry that I uninstalled xDrip but _desperate times calls for desperate measures_ I made sure there was no trace of xDrip on my phone before I reinstall it (deleted all zips, all backups) . I've checked if I had any screenshots prior the problem on Tuesday, but found none. As far as I remember there were no issues related to the battery (the Voltage A and B) since it was a new transmitter recently purchased. 
I've installed the BLE Scanner app on my phone and that's what it found:
![image](https://user-images.githubusercontent.com/42743852/191757905-1935b30a-848c-4e51-92c6-a4977d17effb.png)
None seem to be the Dexcom transmitter.
Thank you for your help! It's much appreciated 👍

**Reply by Navid200** [reply]
You can scroll up and down to see all the devices it has found.  You need to be patient for all to be shown.

it is highly unlikely that a brand new transmitter will just stop working.

**Reply by ailuirazal** [reply]
Hey Navid, 
Sorry for the late reply, I went out to get a backup transmitter from a friend. I've replaced it and in less than 5 minutes it connected to xDrip so I could start the sensor and now I can even see a DexcomLE transmitter on the BLE Scanner app:
![image](https://user-images.githubusercontent.com/42743852/191922866-a4401ef4-d89c-4dca-912e-38b97c57ad3a.png)
As you said it's highly unlikely that a brand new transmitter to go awry but it happened (lucky me! ).
I buy my Dexcom supplies from Czech Republic (as they are cheaper from the Romanian sites): https://eshop.aimport.cz/kat/en/kontinualni-monitorace?country=CZ&currency=EUR
I'll see if I can return it and maybe they'll send it to Dexcom and they could troubleshoot in a less panicked environment ( I'm highly dependent on Dexcom and every time I don't have readings from the sensor my life goes into shambles).
The new transmitter has a different SN but identical dates like the faulty one: 
![image](https://user-images.githubusercontent.com/42743852/191925226-9b4f5171-1a1a-4fab-957f-c919dbeb6588.png)
I'll see how it behaves on the 11th day of functioning.
Thanks again for your help and time! :)
All the best!
Iulia

**Reply by Navid200** [reply]
You're welcome.

The Bluetooth name of a G6 transmitter is Dexcom** where ** is the last two characters of the serial number.
So, what you are seeing in the scanner (DexcomLE) must be one of your old transmitters that still has some battery left and transmitting. 
Your new one will show up as Dexcom5G.

**Reply by ailuirazal** [reply]
Yeap, you're right the new transmitter is Dexcom5G...now it appears in the app:
![image](https://user-images.githubusercontent.com/42743852/191975846-bb0ffff5-17c9-4a54-9df6-fa56d370d46f.png)
Indeed DexcomLE is the previous transmitter I had, started on 25th May 2022 which I replaced with the new one (8ULG9D) which stopped working after 11 days. Funny thing is that it's stored in a cabinet in the living room  :) 
Monday I'll sent the faulty transmitter to Czech Republic, and they'll send it to Dexcom maybe I'll get an official opinion about what happened.
Have a nice weekend!
Iulia
